data.sql

```diff
@@ -41,3 +41,69 @@ VALUES ('Blossom', '1998-10-13', 3, true, 17.0);
 -- Insert data for Ditto
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
 VALUES ('Ditto', '2022-05-14', 4, true, 22.0);
+
+-- Inserting data into owners table
+INSERT INTO owners (full_name, age)
+VALUES ('Sam Smith', 34);
+
+INSERT INTO owners (full_name, age)
+VALUES ('Jennifer Orwell', 19);
+
+INSERT INTO owners (full_name, age)
+VALUES ('Bob', 45);
+
+INSERT INTO owners (full_name, age)
+VALUES ('Melody Pond', 77);
+
+INSERT INTO owners (full_name, age)
+VALUES ('Dean Winchester', 14);
+
+INSERT INTO owners (full_name, age)
+VALUES ('Jodie Whittaker', 38);
+
+SELECT * FROM owners;
+
+-- Insert data into species table
+INSERT INTO species (name)
+VALUES ('Pokemon');
+
+INSERT INTO species (name)
+VALUES ('Digimon');
+
+-- Update animals with names ending in "mon" to have species_id of Digimon
+UPDATE animals
+SET species_id = (SELECT id FROM species WHERE name = 'Digimon')
+WHERE name LIKE '%mon';
+
+-- Update all other animals to have species_id of Pokemon
+UPDATE animals
+SET species_id = (SELECT id FROM species WHERE name = 'Pokemon')
+WHERE species_id IS NULL;
+
+SELECT * FROM animals;
```

Comment:
As requested by the previous reviewer, to maintain proper project structure and organization,  🤓
- Everything that is a query that helps to analyze data should be saved in the `queries.sql `file.


![image](https://github.com/alexansaa/vet_sql/assets/78251927/9469dcc6-8d33-4a82-a692-29c1d521d20b)

Reply:
Please, kindly check again the project requirements:
![image](https://github.com/alexansaa/vet_sql/assets/60579598/eb589645-7a1d-4c68-8e5a-9f5a21606fc4)
As you can see, the required changes don't apply to this code.

Even though the project requirement is requesting otherwise, I have implemented the requirements you are suggesting.

For the next reviewer, please let me pass once and for all because the next review is going to be the last chance I have. Please consider I have made the required changes even though the project requirement is requesting otherwise. Thank you!

Reply:
This change has been done well